webtool/test_transcribe.py

```
@@ -11,6 +11,18 @@
 import transcribe
 
 
+@pytest.fixture(autouse=True)
+def _autocorrect_pin(monkeypatch):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_autocorrect_pin`

Add return type annotation: `None`

(ANN202)